model-optimizer/mo/ops/deformable_convolution.py

```diff
@@ -27,4 +27,9 @@ def __init__(self, graph: Graph, attrs: dict):
     def backend_attrs(self):
         # the same attributes as in a regular convolution and additional attributes 'deformable_group', 'group'
         # and 'bilinear_interpolation_pad'
-        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group', 'bilinear_interpolation_pad']
+        version = self.get_opset()
+        if version == 'opset8':
+            return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group',
+                                                                  'bilinear_interpolation_pad']
+        else:
+            return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group']
```

Comment:
May be a bit cleaner code:

```python3
attrs = ['deformable_group', 'group']
if self.get_opset() == 'opset8':
    attrs.append('bilinear_interpolation_pad')
return attrs
```

Reply:
do we have any relevant test suite to add a test?

Reply:
Unfortunately, no